debian/shadowsocks-rust-local@.service

```diff
@@ -0,0 +1,18 @@
+#  This file is part of shadowsocks-rust.
+#
+#  This is a template unit file. Users may copy and rename the file into
+#  config directories to make new service instances. See systemd.unit(5)
+#  for details.
+
+[Unit]
+Description=Shadowsocks-rust Custom Client Service for %I
+After=network.target
+
+[Service]
+Type=simple
+CapabilityBoundingSet=CAP_NET_BIND_SERVICE
+ExecStart=/usr/bin/sslocal -c /etc/shadowsocks-rust/%i.json
```

Comment:
You should add `--log-without-time` param, because journald already has date infomation.